schemas/context.json

```diff
@@ -10,6 +10,12 @@
 				"nullable"
 			]
 		},
+		"country": {
+			"type": "string",
+			"validation": [
+				"country"
+			]
+		},
```

Comment:
This could be moved into the `pkp-lib` schema so it's not duplicated in each app.